tcf_website/models/models.py

```diff
@@ -795,3 +803,50 @@ class Meta:
                 name='unique vote per user and review',
             )
         ]
+
+
+class DateCreateModMixin(models.Model):
+    """Date create/modified mixin (for posts).
+    """
+    class Meta:
+        abstract = True
+
+    created_date = models.DateTimeField(default=timezone.now)
+    mod_date = models.DateTimeField(blank=True, null=True)
+
+
+class BlogPost(DateCreateModMixin):
+    """Blog post model.
+    https://www.existenceundefined.com/blog/programming/1/how-to-use-django-markdownx-for-your-blog
+    """
+
+    title = models.CharField(max_length=255)
+    subtitle = models.CharField(max_length=255)
+
+    slug = models.SlugField(
+        null=False,
+        unique=True,
+        help_text='The URL that the post will have. Match it to the headline, keep length at 3-5 \
+            words, and use dashes between words. Example: "what-is-url-slug" \
+                (More info: https://rockcontent.com/blog/what-is-url-slug/)')
+    author = models.CharField(max_length=50)
```

Comment:
Is it possible to link the author up with a user model instead of a char field? Might make management easier in the future